bip-0068.mediawiki

```diff
@@ -43,7 +43,7 @@ For time based relative lock-time, 512 second granularity was chosen because bit
 When the relative lock-time is time-based, it is interpreted as a minimum block-time constraint over the input's age. A relative time-based lock-time of zero indicates an input which can be included in any block. More generally, a relative time-based lock-time n can be included into any block produced 512 * n seconds after the mining date of the output it is spending, or any block thereafter.
 The mining date of the output is equals to the median-time-past of the previous block which mined it.
 
-The block produced time is either equals to median time past of its parent or to its nTime field, depending on the state of BIP113 median-time-past.
+The block produced time is equal to the median-time-past of its parent according to BIP113.
```

Comment:
"The block produced time is equal to the median-time-past of its pervious block."